AE_PunktBeschriftung/include/representations/SolutionWithGraph.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <representations/Instance.hpp>
+#include <representations/Point.hpp>
+#include <representations/Solution.hpp>
+
+#include <map>
+
+class SolutionWithGraph : Solution {
```

Comment:
Why is it called SolutionWithGraph? The graph structure is still in the instance, because points know each other, right?

Reply:
Yep